src/braket/ir/jaqcd/instructions.py

```diff
@@ -899,3 +900,43 @@ class Type(str, Enum):
         kraus = "kraus"
 
     type = Type.kraus
+
+
+class StartPreserveBlock(CompilerDirective):
+    """
+    StartPreserveBLock is a compiler instruction to start the code block which
+    will preserve the instruction within StartPreserveBlock and EndPreserveBlock
+    from being removed, reordered, or decompositioned.
```

Comment:
Rewording: "being modified in anyway by the compiler."

Reply:
Updated